src/main/java/com/google/sps/datastore/SetDatastore.java

```diff
@@ -0,0 +1,49 @@
+
+package com.google.sps.datastore;
+
+import java.util.ArrayList;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+//PCAP storage class
+public class SetDatastore {
```

Comment:
Actually combine your Get and Set in the same interface. I was not clear when you mentioned yesterday but the upload and download are basically in the same DAO. Your GenericPCAPDao interface wil have these two methoids and you can implement them in a class.